python_modules/dagster/dagster_tests/core_tests/test_logging.py

```diff
@@ -154,6 +156,60 @@ def test_multiline_logging_complex():
     assert captured_results[0].split("\n") == expected_results
 
 
+def test_handler_in_log_manager():
+    f = io.StringIO()
+    with redirect_stdout(f):
```

Comment:
there's nothing wrong with using this setup, but just for consistency with other tests, you might want to switch this to the capsys pattern you seen in some of the below tests (for example, test_resource_logging()).

Reply:
way cleaner method to capture stdout. modified!